### ISSUE TITLE
Unable to update to  v2.1.0

### ISSUE THREAD

**Issue by PeopleInside** [Nearata/flarum-ext-twofactor#8]
Hi, I upgraded flarum to 1.5.0

but your extension never update

```
 /opt/plesk/php/8.1/bin/php /usr/lib/plesk-9.0/composer.phar update nearata/flarum-ext-twofactor
Loading composer repositories with package information
Updating dependencies
Nothing to modify in lock file
Installing dependencies from lock file (including require-dev)
Nothing to install, update or remove
Package swiftmailer/swiftmailer is abandoned, you should avoid using it. Use symfony/mailer instead.
Generating autoload files
106 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
No security vulnerability advisories found
```

I still have 

```
/opt/plesk/php/8.1/bin/php /usr/lib/plesk-9.0/composer.phar outdated --direct
Color legend:
- patch or minor release available - update recommended
- major release available - update possible
nearata/flarum-ext-twofactor v2.0.1 v2.1.0 Allow your users to enable two factor authentication.
 /opt/plesk/php/8.1/bin/php /usr/lib/plesk-9.0/composer.phar update nearata/flarum-ext-twofactor
```

**Comment by Nearata** [comment]
Hi, should be fixed with release `2.1.1`

**Comment by PeopleInside** [comment]
Yes works @Nearata .
Thank you very much!
